airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that demonstrates interactions with Google Cloud Transfer.
+
+This DAG relies on the following OS environment variables
+
+* GCP_PROJECT_ID - Google Cloud Project to use for the Google Cloud Transfer Service.
+* GCP_DESCRIPTION - Description of transfer job
+* GCP_TRANSFER_SOURCE_AWS_BUCKET - Amazon Web Services Storage bucket from which files are copied.
+* GCP_TRANSFER_FIRST_TARGET_BUCKET - Google Cloud Storage bucket to which files are copied from AWS.
+  It is also a source bucket in next step
+* GCP_TRANSFER_SECOND_TARGET_BUCKET - Google Cloud Storage bucket bucket to which files are copied
+* WAIT_FOR_OPERATION_POKE_INTERVAL - interval of what to check the status of the operation
+
+"""
+import os
+from datetime import datetime, timedelta
+
+from airflow import models
+from airflow.contrib.hooks.gcp_transfer_hook import GcpTransferOperationStatus, GcpTransferJobsStatus
+from airflow.contrib.operators.gcp_transfer_operator import (
+    GcpTransferServiceJobCreateOperator,
+    GcpTransferServiceJobDeleteOperator,
+    GcpTransferServiceJobUpdateOperator,
+    GcpTransferServiceOperationsListOperator,
+    GcpTransferServiceOperationGetOperator,
+    GcpTransferServiceOperationPauseOperator,
+    GcpTransferServiceOperationResumeOperator,
+    GcpTransferServiceOperationCancelOperator,
+)
+from airflow.contrib.sensors.gcp_transfer_sensor import GCPTransferServiceWaitForJobStatusSensor
+from airflow.utils.dates import days_ago
+
+# [START howto_operator_gct_common_variables]
```

Comment:
What are these `START` and `END` notes for?

Reply:
It's is used in guide. Example: https://airflow.readthedocs.io/en/latest/howto/operator.html#google-cloud-functions-operators

It allows you to keep documentation always fresh and working. Our DAGs were launched in our test environment. Every changes in code trigger a job in Cloud Build and run system tests. 
See:  
https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-4+Support+for+System+Tests+for+external+systems
https://github.com/PolideaInternal/airflow-breeze

Reply:
If you would like to start using this environment, I will be happy to help you.